apps/src/storage/levelbuilder/ManifestEditor.jsx

```diff
@@ -1,6 +1,24 @@
 import React from 'react';
 import {connect} from 'react-redux';
 import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+import experiments from '../../util/experiments';
+import LibraryCategory from '../dataBrowser/LibraryCategory';
+
+const styles = {
+  warning: {
+    color: '#9F6000',
+    backgroundColor: color.lighter_yellow,
```

Comment:
Curious whether it makes more sense to use these colors instead (from util/color):

bootstrap_warning_background: #fcf8e3;
bootstrap_warning_text: #c09853;

Reply:
Since those colors aren't part of our core theme either, I don't think it makes a difference one way or the other, and for simplicity I'm inclined to just leave this as is. Let me know if you disagree though!